data/processed_data.csv

```diff
@@ -0,0 +1,161 @@
+Amman,Jordan,361.87,56.68,192.98,254.36,185.02,137.92,328.21,274.2,621.93,1161.75,-895.11
+Pune,India,190.02,22.3,110.81,114.68,41.54,44.06,171.56,81.27,605.44,1311.5,-89.52
+Stockholm,Sweden,1123.73,141.14,297.63,254.72,164.83,122.53,385.95,132.39,2822.83,8050.93,332.3
+Buenos Aires,Argentina,252.03,23.92,127.22,109.58,71.89,110.31,250.98,185.5,551.75,2892.09,-394.17
+Minsk,Belarus,354.44,23.92,131.33,130.74,106.58,71.06,298.35,458.37,494.04,1478.92,-622.36
+San Jose,Costa Rica,494.41,83.58,230.23,174.97,174.74,150.73,316.44,367.73,807.62,1549.32,-817.48
+Casablanca,Morocco,322.88,47.85,138.81,142.04,99.19,81.25,261.41,162.15,462.5,1507.75,-630.93
+Lodz,Poland,366.82,40.3,137.99,117.78,77.83,212.43,241.16,262.53,785.96,1380.19,-408.34
+Montreal,Canada,811.25,108.84,294.17,239.99,77.2,132.37,258.76,513.1,2488.06,3512.15,565.49
+Sao Paulo,Brazil,423.32,72.75,148.93,156.32,173.98,102.04,228.65,436.03,536.08,2217.66,-769.9
+Gothenburg,Sweden,951.05,117.69,285.73,243.73,130.91,96.34,337.82,144.8,2489.14,5266.79,325.87
+Dublin,Ireland,1765.64,188.67,239.72,309.72,118.54,263.26,335.16,1187.3,2885.52,5467.58,-335.18
+Moscow,Russia,801.82,43.48,169.36,151.39,162.55,152.82,373.87,596.57,1106.13,4475.35,-749.17
+Santo Domingo,Dominican Republic,333.34,125.98,157.46,148.06,85.33,123.35,277.26,173.71,382.28,1069.86,-868.5
+Adelaide,Australia,809.17,109.8,234.52,290.43,45.29,194.93,278.24,451.72,3024.72,2392.69,1062.33
+Zurich,Switzerland,1690.31,231.86,646.38,367.58,237.04,250.09,497.05,2501.89,6514.87,10719.61,2594.55
+Yerevan,Armenia,279.64,19.35,121.8,103.28,100.75,85.56,264.51,161.49,337.05,1195.64,-637.84
+Manila,Philippines,424.47,21.94,146.88,101.05,74.52,194.42,247.13,164.09,408.35,1973.73,-802.06
+Brisbane,Australia,1025.49,153.25,237.07,311.77,105.6,183.52,298.6,1224.77,3454.23,3792.51,1138.94
+Jakarta,Indonesia,356.83,24.29,177.16,205.52,85.56,143.86,248.7,160.11,472.34,1882.27,-769.57
+Ankara,Turkey,184.78,41.86,136.75,110.31,67.04,82.1,185.38,276.51,501.79,634.3,-306.43
+Lviv,Ukraine,350.29,15.31,122.25,85.5,59.87,106.92,322.52,219.46,389.17,1021.36,-673.49
+Novosibirsk,Russia,324.89,17.68,147.11,107.48,110.27,117.84,312.29,254.03,497.36,1175.4,-640.2
+Bursa,Turkey,170.58,43.26,145.14,108.94,64.11,95.75,172.12,228.58,505.99,546.91,-293.92
+Brussels,Belgium,868.9,122.55,292.16,242.32,107.01,197.57,343.32,792.8,2610.25,3289.04,436.41
+Jerusalem,Israel,945.89,95.45,269.63,308.39,156.94,222.23,401.27,645.45,2178.68,7229.86,-221.13
+Melbourne,Australia,1136.76,146.67,239.26,304.24,117.24,199.21,293.88,1425.47,3182.83,6138.7,745.56
+Perth,Australia,876.12,132.38,234.2,309.12,99.5,207.94,288.56,1356.74,2842.9,3917.46,695.08
+Sydney,Australia,1544.16,195.61,257.28,327.36,115.56,188.8,320.08,1583.95,3440.93,7872.15,492.08
+Alexandria,Egypt,129.21,14.41,114.68,100.91,56.61,40.39,270.05,63.17,186.89,549.26,-539.39
+Quito,Ecuador,336.85,73.31,159.55,151.16,122.93,90.87,308.65,315.36,600.99,1331.22,-642.34
```

Comment:
Hi @hjw0703, 

Great work so far on the preprocessing, please review my comment [here](https://github.com/UBC-MDS/Cost_of_living_py/issues/4) in the Data Processing issue, would request you to consider this suggestion. Thank you!

Reply:
Thanks for your advice, as car prices vary a lot between different makes and models, I added car price for a popular model in the world: Toyota corolla 1.6L for our future visualization.